src/helpers.js

```diff
@@ -2,40 +2,73 @@ const fs = require('fs');
 const path = require('path');
 
 /**
- * 
+ *
  * @typedef {Object} AbellConfigs - Configurations from abell.config.js
  * @property {String} contentPath - Path to content directory (default 'content')
  * @property {String} sourcePath - Path to source directory (default 'src')
  * @property {String} destinationPath
  *  - Path to output destination (default 'dist', changes to 'debug' during dev-server)
  * @property {Object} globalMeta - Meta variables that are accessible globally in .abell files
- * 
+ *
  */
 
+const relativeJoinedPath = (pathString) =>
+  path.join(process.cwd(), ...pathString.split('/'));
 
-const relativeJoinedPath = pathString => path.join(process.cwd(), ...pathString.split('/'));
-
-const getDirectories = source => {
-  return fs.readdirSync(source)
-    .filter(dirent => {
-      return fs.lstatSync(path.join(source, dirent)).isDirectory();
-    });
+const getDirectories = (source) => {
+  return fs.readdirSync(source).filter((dirent) => {
+    return fs.lstatSync(path.join(source, dirent)).isDirectory();
+  });
 };
- 
+
 const rmdirRecursiveSync = function(pathToRemove) {
   if (fs.existsSync(pathToRemove)) {
     fs.readdirSync(pathToRemove).forEach((file, index) => {
       const curPath = path.join(pathToRemove, file);
-      if (fs.lstatSync(curPath).isDirectory()) { // recurse
+      if (fs.lstatSync(curPath).isDirectory()) {
+        // recurse
         rmdirRecursiveSync(curPath);
-      } else { // delete file
+      } else {
+        // delete file
         fs.unlinkSync(curPath);
       }
     });
     fs.rmdirSync(pathToRemove);
   }
 };
 
+const recFindByExt = (base, ext, inputFiles, inputResult) => {
+  const files = inputFiles || fs.readdirSync(base);
+  let result = inputResult || [];
+
+  files.map(function(file) {
```

Comment:
This too can be done with `for (const file of files) {}`

Reply:
Fixed this ✅